### DISCUSSION TITLE
To do copy errors

### DISCUSSION THREAD

**Discussion by Hy-Ascension** [AvaloniaUI/Avalonia.Samples#104] [Q&A] [answered]
Im new to programming and avalonia and was just copying the to do list to try to understand how everything interacts, but i'm getting a few errors
![image](https://github.com/user-attachments/assets/264d992a-5c57-4001-8746-4cfb3e858eb0)
i'll still look through the code to see how its supposed to interact but if i can get help to actually get the exmple working that would be great.
I'm also going to write summaries for each line of code

**Comment by timunie** [comment, marked as answer]
@Hy-Ascension I've never seen something like this. Can you tell me what you did so far? Did you clone the source or are you trying to create on your own? If so, do you have the latest Avalonia templates installed?

**Reply by Hy-Ascension** [reply]
@timunie thanks for the reply i didnt realise i could clone the repository that worked.
